### PR TITLE
Content review

### PR DESCRIPTION
**Pull request by natewcl**
@julieritchie @kellyfinn Please review it

### CODE REVIEW

**Comment by jprovine** [comment]
Will this be pushed today?

On Thu, Mar 12, 2015 at 5:01 PM, Nate Leung <notifications@github.com>
wrote:

> @julieritchie <https://github.com/julieritchie> @kellyfinn
> <https://github.com/kellyfinn> Please review it
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/optimizely/marketing-website/pull/914
> Commit Summary
>
>    - Create sessioncam/index.hbs
>
> File Changes
>
>    - *A* website/partners/technology/sessioncam/index.hbs
>    <https://github.com/optimizely/marketing-website/pull/914/files#diff-0>
>    (37)
>
> Patch Links:
>
>    - https://github.com/optimizely/marketing-website/pull/914.patch
>    - https://github.com/optimizely/marketing-website/pull/914.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/optimizely/marketing-website/pull/914>.
>



-- 
*John Provine*
Product Manager, Optimizely
john@optimizely.com | 917.756.3717

**Comment by julieritchie** [comment]
@jprovine we're working on a weekly sla for these now

**Comment by jprovine** [comment]
Hey Julie, a weekly SLA is a huge bottleneck for a small change to update
the partner directory and will hurt our momentum.  Is this a site-wide
policy or just the partner directory?

On Fri, Mar 13, 2015 at 7:14 PM, Julie Ritchie <notifications@github.com>
wrote:

> @jprovine <https://github.com/jprovine> we're working on a weekly sla for
> these now
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/optimizely/marketing-website/pull/914#issuecomment-79705772>
> .
>



-- 
*John Provine*
Product Manager, Optimizely
john@optimizely.com | 917.756.3717

**Comment by dtothefp** [comment]
@julieritchie @jprovine says this should be merged ASAP, there is still no logo and a big line where there should be `kb_article`, could you please advise?

**Comment by julieritchie** [comment]
@jprovine can you set up a meeting to discuss? I think I should clarify better what our process it and why it's important

**Comment by kellyfinn** [comment]
This is ready @julieritchie @dtothefp @stewsmith

**Comment by dtothefp** [comment]
retest this please

**Comment by natewcl** [comment]
@julieritchie @kellyfinn @jprovine Can't find the index.hbs file to edit KB article link

Can one of you please insert this link: 
https://help.optimizely.com/hc/en-us/articles/204333128

File: website/partners/technology/sessioncam/index.hbs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+layout: partners.hbs
+title: "SessionCam"
+integrations: true
+priority: 60
+summary: "Watch how customers interact with each test to get an unparalleled level of insight into the effectiveness of each test."
+logo: "https://s3.amazonaws.com/cloud_cannon_production/optimizely-marketer-assets.cloudvent.net/raw/partner-logos/technology/sessioncam.png?timestamp=38adbfc56f80bf9ec12814395bf73394"
+website_link: "http://www.sessioncam.com/"
+website_display: "www.sessioncam.com"
+kb_article: "_____"
+stars: 0
```
Comment (dtothefp) [inline website/partners/technology/sessioncam/index.hbs]: @kellyfinn this isn't ready, please ensure that the `kb_article` does not have strange text in it and remove `http:` or `https:` protocol from logo

**Comment by dtothefp** [comment]
@natewcl link added...LGTM

**Pull request merged**